src/main.rs

```diff
@@ -157,6 +160,14 @@ pub enum Command {
         /// Name of the state to get
         name: String,
     },
+    /// Creating comment based on issues and pull requests
+    PostComment {
+        /// The content of comment message
+        message: String,
+        /// The secret name for authorization. GITHUB_TOKEN is used by default.
+        #[structopt(short="t", long="token", default_value = "GITHUB_TOKEN")]
```

Comment:
```suggestion
        #[structopt(short="t", long="token", env = "GITHUB_TOKEN")]
```
so that it loads the token as an environment variable by default